Source/Engine/Physics/Physics.h

```diff
@@ -88,41 +90,44 @@ DECLARE_SCRIPTING_TYPE_NO_SPAWN(Physics);
 #endif
 
     /// <summary>
-    /// Gets PhysX scene object
+    /// Gets the physics tolerances scale.
     /// </summary>
-    static PxScene* GetScene();
+    static PxTolerancesScale* GetTolerancesScale();
 
     /// <summary>
-    /// Gets PhysX characters controller manager object
+    /// Gets the default physical material.
     /// </summary>
-    static PxControllerManager* GetControllerManager();
+    static PxMaterial* GetDefaultMaterial();
 
+public:
+    
     /// <summary>
-    /// Gets the physics tolerances scale.
+    /// The default physics scene.
     /// </summary>
-    static PxTolerancesScale* GetTolerancesScale();
+    API_FIELD(ReadOnly)
+    static PhysicsScene* DefaultScene;
 
     /// <summary>
-    /// Gets the default query filter callback used for the scene queries.
+    /// List with all physics scenes (readonly).
     /// </summary>
-    static PxQueryFilterCallback* GetQueryFilterCallback();
+    API_FIELD(ReadOnly)
+    static Array<PhysicsScene*> Scenes;
```

Comment:
Can we use forward declaration `Array<PhysicsScene*, HeapAllocation>` and remove include to Array in `Physics.h`?

Reply:
Done